README.md

```diff
@@ -65,6 +65,7 @@ docker run \
   -p 5432:5432 \
   -d tensorchord/vchord-postgres:pg17-v0.2.2
 ```
+> In addition to the base image with the VectorChord extension, we provide an all-in-one image, `tensorchord/vchord-suite:pg17-latest`. This comprehensive image includes all official TensorChord extensions. Developers should select an image tag that is compatible with their extension's version, as indicated in [the support matrix](https://github.com/tensorchord/VectorChord-images?tab=readme-ov-file#support-matrix).
```

Comment:
Use > [!NOTE]
And clearly state BM25 extension here

Reply:
Like this? > BM25 extension is currently in alpha stage and not recommended for production use. We're still iterating on the tokenizer API to support more configurations and languages. The interface may change in the future.

Reply:
updated, @VoVAllen .